lib/frontapp/client.rb

```diff
@@ -56,11 +56,9 @@ def list(path, params = {})
       query = format_query(params)
       url = "#{base_url}#{path}?#{query}"
       until last_page
-        res = @headers.get(url)
-        if !res.status.success?
-          raise Error.from_response(res)
-        end
-        response = JSON.parse(res.to_s)
+        res = @connection.get(url)
+        raise Error.from_response(res) unless res.status == 200
```

Comment:
(res.status / 100.0).floor == 2

Reply:
actually, there appears to be a `success?` helper https://lostisland.github.io/faraday/#/getting-started/env-object?id=helpers

Reply:
I tried locally, but it did not work, let me have a look

Reply:
ok... found the helper